.travis.yml

```diff
@@ -12,23 +12,28 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-cache: "pip"
-sudo: "false"
-dist: "trusty"
-language:
-  - "python"
+sudo: required
```

Comment:
Do we need this? This doesn't allow us to use the [faster](https://docs.travis-ci.com/user/ci-environment/#Virtualization-environments) host machine-type with Travis.